.tekton/main-pipeline.yaml

```diff
@@ -175,79 +160,162 @@ spec:
       operator: in
       values: [ "true" ]
     workspaces:
-    - name: output
-      workspace: workspace
     - name: basic-auth
       workspace: git-auth
 
   - name: determine-image-tag
     params:
-    - name: tag-suffix
+    - name: TAG_SUFFIX
       value: $(params.output-tag-suffix)
-    runAfter:
-    # This task must run on a freshly cloned repository to prevent seeing any changes from other tasks.
-    - clone-repository
+    - name: SOURCE_ARTIFACT
+      value: $(tasks.clone-repository.results.SOURCE_ARTIFACT)
     taskRef:
-      name: determine-image-tag
-    workspaces:
-    - name: source
-      workspace: workspace
+      name: determine-image-tag-oci-ta
 
   - name: copy-subscription-manager-activation-key
-    runAfter:
-    - determine-image-tag
+    params:
+      - name: SOURCE_ARTIFACT
+        value: $(tasks.clone-repository.results.SOURCE_ARTIFACT)
+      - name: ociStorage
+        value: $(params.output-image-repo):konflux-$(params.revision).subscription-manager
+      - name: ociArtifactExpiresAfter
+        value: $(params.oci-artifact-expires-after)
     workspaces:
-    - name: source
-      workspace: workspace
     - name: subscription-manager-activation-key
       workspace: subscription-manager-activation-key
     taskSpec:
+      params:
+      - name: SOURCE_ARTIFACT
+        description: The Trusted Artifact URI pointing to the artifact with
+          the application source code. This should be the result of the git-clone task,
+          results from other tasks might fail as dirty.
+        type: string
+      - name: ociStorage
+        description: The OCI repository where the Trusted Artifacts are stored.
+        type: string
+      - name: ociArtifactExpiresAfter
+        description: Expiration date for the trusted artifacts created in the
+          OCI repository. An empty string means the artifacts do not expire.
+        type: string
+        default: ""
+      results:
+      - name: SOURCE_ARTIFACT
+        description: The Trusted Artifact URI pointing to the artifact with
+          the smuggled activation key.
```

Comment:
```suggestion
        description: The Trusted Artifact URI pointing to the application source code with
          the additional smuggled activation key.
```

Reply:
Done in 15566365d11efcbb1d8563313489922ed725117a